src/wp-admin/async-upload.php

```diff
@@ -132,6 +134,7 @@
 			'paragraph_wrap'     => false,
 		)
 	);
+	echo "<script type='text/javascript'>jQuery( 'button#" . esc_js( $button_unique_id ) . "' ).on( 'click', function() {jQuery(this).parents('div.media-item').slideUp(200, function(){jQuery(this).remove();})});</script>\n";
```

Comment:
Strictly speaking, the `type` attribute is not recommended when the `script` element contains JavaScript, as that's the default for the script element in HTML5. Since the WordPress admin uses HTML5, I think we should start omitting these attributes.

Reply:
> Since the WordPress admin uses HTML5, I think we should start omitting these attributes.

Removed in c7007ea5bd9bf9b5d03467701fd820e470f8d119.

> There should also be an announcement when the error is thrown, but I think that's outside the scope of this issue. Adding an announcement about the dismissal, however, is pretty easy in this context.

I'm happy to add this in a follow up, I'd need your advice on what the announcement should be to account for the fact there can be multiple dismissable messages for different files.